lib/src/test/java/com/auth0/android/lock/OptionsTest.java

```diff
@@ -94,23 +107,20 @@ public void shouldNotSetPrivacyPolicyURLWhenInvalidURL() {
     }
 
     @Test
-    public void shouldSetTermsOfServiceURL() {
-        Options options = new Options();
-        options.setAccount(auth0);
+    public void shouldSetTermsOfServiceURL() throws Exception {
         options.setTermsURL("https://valid.url/terms");
 
         Parcel parcel = Parcel.obtain();
         options.writeToParcel(parcel, 0);
         parcel.setDataPosition(0);
 
         Options parceledOptions = Options.CREATOR.createFromParcel(parcel);
+        assertThat(options.getTermsURL(), is(notNullValue()));
```

Comment:
what if `options.getTermsURL()` return "some other url that I'm mixing with privacy url for example" ?